patches/0072-WIFI-13428-Fix-Traffic-Disruption-for-MPSK-clients-f.patch

```diff
@@ -0,0 +1,111 @@
+From da1d005199f6485c37aed3ae15faf1b9ab79d88d Mon Sep 17 00:00:00 2001
+From: joydeepbenison <joydeep.ghosh@benisontech.com>
+Date: Mon, 27 May 2024 13:34:25 +0530
+Subject: [PATCH] WIFI-13428 Fix:Traffic Disruption for MPSK clients for
+ yuncore_fap655
+
+Fix Traffic Disruption for MPSK clients for yuncore_fap655 when
+vlan-id of the lan port is modified.Any change in swconfig resets the
+internal switch of FAP655 due to which all the dynamically added vlans
+for MPSK clients are lost. Configure the Dynamic Vlans for MPSK Clients
+again after switch reset.
+
+Signed-off-by: joydeepbenison <joydeep.ghosh@benisontech.com>
+---
+ .../files/etc/hotplug.d/net/50-switch-dvlan   | 43 +++++++++++++++++++
+ .../files/lib/functions/dyn_vlan_switch.sh    | 33 ++++++++++++++
+ 2 files changed, 76 insertions(+)
+ create mode 100644 package/base-files/files/etc/hotplug.d/net/50-switch-dvlan
+ create mode 100644 package/base-files/files/lib/functions/dyn_vlan_switch.sh
+
+diff --git a/package/base-files/files/etc/hotplug.d/net/50-switch-dvlan b/package/base-files/files/etc/hotplug.d/net/50-switch-dvlan
+new file mode 100644
+index 0000000000..f24329cfe6
+--- /dev/null
++++ b/package/base-files/files/etc/hotplug.d/net/50-switch-dvlan
+@@ -0,0 +1,43 @@
++#!/bin/sh
++
++. /lib/functions.sh
++. /lib/functions/dyn_vlan_switch.sh
++
++debug() {
++	logger -t HOTPLUG-DVLAN "$*"
++}
++
++if [ "${INTERFACE:0:2}" != "up" ]; then
++	exit 0
++fi
++
++case "$(board_name)" in
```

Comment:
I'd really prefer if we can keep per-model config like this in a central place, rather than having various parts of the system that need updating upon adding new models.    I believe all this info is available in `/etc/board.json` and, to some degree, in `/etc/config/network/` and `/etc/config/event` -- I'd suggest trying to read it from `board.json` as the single source of truth though.   What do you think?

Here's snippet from `board.json` of WF-186W

```
	"switch": {
		"switch0": {
			"enable": true,
			"reset": true,
			"ports": [
				{
					"num": 4,
					"role": "wan"
				},
				{
					"num": 0,
					"role": "lan"
				},
				{
					"num": 1,
					"role": "lan"
				},
				{
					"num": 2,
					"role": "lan"
				},
				{
					"num": 3,
					"role": "lan"
				},
				{
					"num": 6,
					"device": "eth0",
					"need_tag": false,
					"want_untag": false
				}
			],
```

Reply:
@phwhite patch imho uses the right place in the chain of events to fix the issue, yet the code is not optimal. I plan to clean this up tomorrow.